projects/libs/core-library-typescript/package.json

```diff
@@ -57,9 +57,11 @@
     "@types/winston": "^2.3.7",
     "@types/ws": "^4.0.0",
     "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
```

Comment:
Could you add `@types/chai-as-promised` and `@types/sinon-chai` (or are they already included with something else?)

Reply:
shit, I forgot to port them from the other PR